docs/sources/http_api/short_url.md

```diff
@@ -0,0 +1,77 @@
++++
+title = "ShortUrl HTTP API "
+description = "Grafana ShortUrl HTTP API"
+keywords = ["grafana", "http", "documentation", "api", "shortUrl"]
+aliases = ["/docs/grafana/latest/http_api/short_url/"]
+type = "docs"
+[menu.docs]
+name = "ShortUrl"
+parent = "http_api"
++++
+
+# ShortUrl API
```

Comment:
Please review the [Grafana documentation style guide](https://github.com/grafana/grafana/blob/master/contribute/style-guides/documentation-style-guide.md).

At a glance, please apply the following:
- Headings use sentence case.
- URL is always in all caps. Same for ID and UID.
- Put periods at the end of sentences.
- Include punctuation in a word's formatting.

Reply:
i think i got everything!